bats/tests/extensions/containers.bats

```diff
@@ -80,6 +80,14 @@ encoded_id() { # variant
     assert_line --regexp "$(id vm-compose).*[[:space:]]Up[[:space:]]"
 }
 
+@test 'compose - check for dangling symlinks' {
+    if ! using_containerd; then
+        skip 'This test only applies to containerd'
```

Comment:
Why does this test not apply to `moby`?

Aren't we trying for identical behaviour?

If the moby behaviour is different, shouldn't we have a test to verify that expected behaviour here?

Reply:
Yeah, there's still a behaviour difference here (docker actually extracts symlinks as symlinks, dangling or not). Except on Windows without admin (where they're not allowed to create symlinks); in that case the symlinks are just silently ignored.

This will require a better fix.